packages/modular-scripts/types/package/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "PackageName__",
   "version": "1.0.0",
-  "main": "index.ts",
+  "main": "src/index.ts",
```

Comment:
Would removing `*/src/**/*` from the RegEx not fix this also? Or do we need a point of reference `src` to avoid running unexpected tests?

Reply:
We could... but that would mean we have to change our typescript matches as well. Which could be fine, but that'll complicate making builds later, because it'll be hard to isolate built artifacts. I think it's better to isolate this stuff in a folder.